ingestion/src/main/java/feast/store/serving/redis/FeatureRowToRedisMutationDoFn.java

```diff
@@ -64,14 +64,20 @@ private RedisKey getKey(FeatureRow featureRow) {
     return redisKeyBuilder.build();
   }
 
+  private byte[] getValue(FeatureRow featureRow) {
+    FeatureRowEncoder encoder =
+        new FeatureRowEncoder(featureSets.get(featureRow.getFeatureSet()).getSpec());
+    return encoder.encode(featureRow).toByteArray();
```

Comment:
Would this make more sense as a static method `encode(featureRow, featureSet)`?

Reply:
Yup, a static method does make more sense. Unless we need to support multiple encoding mechanism, which is unlikely.